oaknational/Oak-Web-Application
# [PUPIL-800] Lesson overview and lesson review colour updates

## Description

Music year: {owa_music_year}

Updates Oak components to bring in new background colours and background svgs.
Updates the Overview and Review areas to change colour based on phase
Updates the quizzes to show a svg background when the question has been answered correctly.

[Ticket here](https://www.notion.so/oaknationalacademy/OakComponents-Lesson-overview-Lesson-review-colour-updates-065f72d47fdb4b7a9d808b20a87bcb5a) and svg ticket [here](https://www.notion.so/oaknationalacademy/OakComponents-OakLessonLayout-SVG-Illustration-applied-to-desktop-page-backgrounds-03b172a946894d47b3c8077971816d37)

## How to test

1. Go to https://deploy-preview-2639--oak-web-application.netlify.thenational.academy/pupils/programmes/design-technology-primary-year-2-l/units/mechanisms-sliders-and-levers-8ef2/lessons/to-explore-a-range-of-sliders-and-levers-69jkgc/overview
2. You should see a **primary** background colour, svg and subject icon
3. Go to introduction, you should see the background colour/svg match the button
4. Go back and go to the lesson video, you should see the background colour/svg match the button

_

1. Go to https://deploy-preview-2639--oak-web-application.netlify.thenational.academy/pupils/programmes/design-technology-primary-year-2-l/units/mechanisms-sliders-and-levers-8ef2/lessons/to-explore-a-range-of-sliders-and-levers-69jkgc/review
2. You should see a **primary** background colour and svg

_

1. Go to https://deploy-preview-2639--oak-web-application.netlify.thenational.academy/pupils/programmes/computing-secondary-year-9-l/units/animations-fd00/lessons/move-rotate-scale-colour-6grp6c/overview
2. You should see a **secondary** background colour, svg and subject icon
3. Go to introduction, you should see the background colour/svg match the button
4. Go back and go to the lesson video, you should see the background colour/svg match the button

_

1. Go to https://deploy-preview-2639--oak-web-application.netlify.thenational.academy/pupils/programmes/computing-secondary-year-9-l/units/animations-fd00/lessons/move-rotate-scale-colour-6grp6c/exit-quiz
2. You should see a yellow background with no svg
3. When you get a question correct you should see the svg confetti background
4. When you get a question wrong you should **not** see the svg confetti background

_

On mobile the backgrounds are not visible.

## Screenshots

![lesson colour changes](https://github.com/user-attachments/assets/b74a51a0-e722-42f1-810b-a95837b591fb)

## Checklist
